quiz.js

```diff
@@ -47,40 +47,53 @@ function check() {
     resulthead.innerHTML="Combination Skin Type";
     result.innerHTML="So what does this mean?" + "<br>"
     + "<br>" + "It is very likely that you have combination type skin. This means that you have oily skin in some areas of your face (typically the T-zone) and dry skin in other areas. We recommend that you use products for combination type skin that is gentle enough for dry skin but strong enough to remove excess oils from your face.";
-
-    }
+    var str="To combination skin type products page";

```

Comment:
I changed the links that would be at the bottom of the user's result screen based on the skin type that they get. So in this instance, they would be redirected to "products-3" which will contain products for combination type skin.